nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/service/TestS3ColdStorageService.java

```diff
@@ -48,27 +47,18 @@
 public class TestS3ColdStorageService extends AbstractTestColdStorageService {
 
     @Test
-    public void shouldMoveToColdStorageIfBlobAlreadyIs() throws IOException {
+    public void shouldRestoreFromColdStorageWhenReuploaded() throws IOException {
```

Comment:
Not sure to understand if don't need to test the previous case: MoveToColdStorageIfBlobAlreadyIs, or if the new tests covers both scenarios : MoveToColdStorageIfBlobAlreadyIs and RestoreFromColdStorageWhenReuploaded?

Reply:
With this commit platform side https://github.com/nuxeo/nuxeo-lts/commit/ac7e0caafd89a70f3cddc17bfab78ea1f8d2e295 the previous version of the test (MoveToColdStorageIfBlobAlreadyIs) won't pass anymore because expected behavior has changed. when creating a document referencing an existing blob already in cold storage, the newly created doc was automatically moved to Cold Storage (in terms of facets and schema). Now, we instead restore the blob (by overwriting it in the store) and we rather restore the other documents referencing it (in terms of facets and schema)